codalab/model/bundle_model.py

```diff
@@ -742,6 +742,11 @@ def update_bundle(self, bundle, update, connection=None):
         # Apply the column and metadata updates in memory and validate the result.
         metadata_update = update.pop('metadata', {})
         bundle.update_in_memory(update)
+
+        # Make individual tags out of space separated words
+        if metadata_update.has_key('tags'):
+            metadata_update['tags'] = metadata_update['tags'].split()
+
```

Comment:
Hi @kashizui:
Please have a look, I think worksheet should work, but I am a little confused about the bundle tags as they were not explicitly mentioned like worksheet. I did some back tracing and checked out bundle metadata schema. I had an intuition that this method is the point where manipulation done, would set things right further. I'm not completely certain.